File: Source/Templates/NetCore/GtkSharp.Template.CSharp/content/GtkSharp.Window.CSharp/Gtk_Window.cs
using System;
using Gtk;
using UI = Gtk.Builder.ObjectAttribute;

namespace GtkNamespace
{
    class Gtk_Window : Window
    {
        public Gtk_Window() : this(new Builder("Gtk_Window.glade")) { }

        private Gtk_Window(Builder builder) : base(builder.GetObject("Gtk_Window").Handle)
        {
            builder.Autoconnect(this);
        }
    }
}

File: Source/Templates/GtkSharp.Template.VBNet/content/GtkSharp.Window.VBNet/Gtk_Window.vb
Imports System
Imports Gtk
Imports UI = Gtk.Builder.ObjectAttribute

Namespace GtkNamespace
    Public Class Gtk_Window
        Inherits Window

        Public Sub New (builder as Builder)
            MyBase.New (builder.GetObject("Gtk_Window").Handle)

            builder.Autoconnect (Me)
        End Sub
        
        Public Sub New ()
            Me.New (new Builder ("Gtk_Window.glade"))
        End Sub
        
    End Class
End Namespace
